backend/routes/FormRoutes.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+const formController = require('../controllers/formController.js')
+
+const router = express.Router();
+
+// router.post("/createform",formController.createForm);
+// Create a new form
+router.post('/createform', async (req, res) => {
+    const { eventId, description,price, data } = req.body;
+    try {
+        const newForm = await formController.createForm(eventId, description,price, data);
+        res.status(201).json(newForm);
+    } catch (error) {
+        res.status(500).json({ error: error.message });
+    }
+});
```

Comment:
Don't put controller logic in the router. Keep all the functions in the controller